py/desitarget/io.py

```diff
@@ -2208,3 +2217,13 @@ def _check_hpx_length(hpxlist, length=68, warning=False):
         else:
             log.critical(msg)
             raise ValueError(msg)
+
+
+def _check_both_set(hpxlist, nside):
```

Comment:
Normally an `_underscore()` function is hidden and used only within the file that defines it, but this function seems useful enough that you are importing it in multiple bin/ scripts.  Let's upgrade it to a non-underscore function.

Reply:
Done. I fooled myself by making it local to `io.py` and then adopting it in basically every single executable in the `bin` directory! Bad form.